vpr/src/base/vpr_context.h

```diff
@@ -288,6 +288,11 @@ struct RoutingContext : public Context {
 
     std::vector<t_rr_node_route_inf> rr_node_route_inf; /* [0..device_ctx.num_rr_nodes-1] */
 
+    //Information about whether a node is a non-configurable. 1 bit for each node
```

Comment:
Reword: node is part of a non-configurable set (i.e. connected to others with non-configurable edges like metal shorts that can't be disabled).